cob_generic_can/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_generic_can)
 
-find_package(catkin REQUIRED COMPONENTS cob_utilities libntcan libpcan socketcan_interface)
+find_package(catkin REQUIRED COMPONENTS cmake_modules cob_utilities libntcan libpcan socketcan_interface)
```

Comment:
You don't need `cmake_modules` for that. In fact, [cmake_modules](https://github.com/ros/cmake_modules/tree/0.4-devel/cmake/Modules) has not entries for Boost!